src/logic/function/basic.lean

```diff
@@ -92,6 +92,12 @@ lemma injective.of_comp_iff' (f : α → β) {g : γ → α} (hg : bijective g)
     hx ▸ hy ▸ λ hf, h hf ▸ rfl,
   λ h, h.comp hg.injective⟩
 
+
```

Comment:
```suggestion
```

Reply:
Although I extracted this to #10406, since touching a file this low in the heirarchy is going to make the builds tediously slow here.